components/admin/requests/reason-for-replacement/Form.tsx

```diff
@@ -54,13 +54,21 @@ export default function ReasonForReplacementForm({
               {'Stolen'}
             </Radio>
             <Radio
-              value={'OTHER'}
+              value={'MAIL_LOST'}
               onChange={() => {
                 setFieldValue('paymentInformation.processingFee', '0');
               }}
             >
               {'Mail Lost'}
             </Radio>
+            <Radio
+              value={'OTHER'}
```

Comment:
Just occurred to me that we need to discuss how to handle existing replacement applications that were created after #325, which mapped the `OTHER` enum to the displayed value of 'Mail Lost'. Once this is PR merged, any application using `OTHER` will show as 'Other' even if they were supposed to be 'Mail Lost'.

Let's discuss in Wednesday's meeting, can you please hold off on merging this PR (at least the frontend portion)?